kotlin/internal/utils/sets.bzl

```diff
@@ -0,0 +1,79 @@
+#
```

Comment:
Why not use skylib? https://github.com/bazelbuild/bazel-skylib/blob/master/docs/new_sets_doc.md

Reply:
I had some issues when I did so on bazel_maven_repository, partly in terms of being over-wrought for what I needed, having a separate whole struct wrapping the inner dict. But also adding an extra workspace dep is pretty rough in a bazel world. They don't have good ways of handling and identifying diamond dependency versioning problems, so you just "get what you get" and no way to validate that the version of skylib I"m using here, isn't going to mess with the version of skylib pushed by the maven integration, etc. etc.  This is just cleaner.  If we find we need it anyway, then yeah, rip this out, but for now it seemed just much cleaner.